packages/client3/pages/api/accounts/accounts.json

```diff
@@ -1 +1 @@
-{"accounts":[{"wallet":"0x108Cc3833CD49333A7908e4bB52f4CF8f4090425","role":"Artist"}]}
\ No newline at end of file
+{"accounts":[{"wallet":"0x108Cc3833CD49333A7908e4bB52f4CF8f4090425","role":"Artist"},{"wallet":"0x062765d960DEa3e15cdeB8313ECE1b556FE23FCf","role":"Artist"}]}
```

Comment:
Whoops, we can take these out. Maybe add to the .gitignore?

Reply:
I think we gonna need to keep this information just like the metadata from pools